Fix/cms app test

nikrb opened the pull request:
fix app test.
Moved article list fetch to CMSContainer.
Factor out api fetches to actions file.
Simplify Sidebar and added expand on article refresh/bookmark whilst I was here

nikrb committed:
refactor article fetch into cms container

nikrb committed:
merge develop

nikrb committed:
fix: cms container actions export

nikrb committed:
remove auth from BE articles fetch

cms routes pass everything back and we don't want content for sidebar pop
so used old test article fetch

nikrb committed:
update sidebar tree generation to expand on reload/bookmark

nikrb committed:
factor article fetch out of content area

nikrb committed:
add open attrib to details to expand cms tree on refresh

nikrb committed:
fix: articles prop name

nikrb committed:
simplify sidebar

nikrb committed:
remove debug log

nikrb committed:
test: update snapshot

nikrb committed:
fix: linting

jenovs commented on frontend/src/CMSContainer/CMSContainer.js:
```diff
@@ -1,40 +1,45 @@
+    return (
+      <Wrapper>
+        <BrowserRouter>
+          <React.Fragment>
```
`<BrowserRouter>` (and `<React.Fragment>`) are unnecessary.

jenovs commented on frontend/src/Sidebar/Sidebar.js:
```diff
@@ -4,59 +4,46 @@ import { getTree, getChildren } from './utils';
+  };
+
+  static defaultProps = {
+    match: '',
```
propTypes is object

nikrb committed:
remove additional BrowserRouter in CMSContainer

This also fixes the issue clicking on navbar cms when viewing a cms/<articleId>

nikrb committed:
use conditional rendering in content area instead of keeping component in state

nikrb committed:
review: fix Sidebar default match prop type and fix badly named variable

nikrb committed:
review: mark sidebar match prop required
